docs/functions/SDDC (Cloud Builder)/Get-CloudBuilderSDDC.md

```diff
@@ -0,0 +1,46 @@
+# Get-CloudBuilderSDDC
```

Comment:
Intentionally not using Get-VCFCloudbuilder?

Reply:
Yes - I thought about using VCF in front but decided to not add it - Wanted to make it clear the cmdlts were different endpoints and this seemed like the best approach

Reply:
i think, it is always better to kept -VCFXXX, you known it is coming from PowerVCF...  (but only my opinion !)